src/ProductAttributePageTitle.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * Allows optional customization of page title.
+ */
+class ProductAttributePageTitle {
+
+  public static function init() {
+    $attribute_terms = wc_get_attribute_taxonomy_names();
+    $group_filter = [];
+
+    foreach( $attribute_terms as $attribute_term ) {
+      $group_filter[] = [[
+        'param'    => 'taxonomy',
+        'operator' => '==',
+        'value'    => $attribute_term,
+      ]];
+    }
+
+    // Register ACF field.
+    ProductAttributePageTitle::register_acf_page_title($group_filter);
```

Comment:
Every frontend page view loads all attribute taxonomies now, even though the frontend may not need the data.

Maybe you can move this block into a new method `admin_init()` in this class and only invoke it (from elsewhere) when `is_admin()` is true?